uwblueprint/starter-code-v2
# Add annotations and test scrubber

## Notion ticket link

[Ticket Name](https://www.notion.so/uwblueprintexecs/Task-Board-db95cd7b93f245f78ee85e3a8a6a316d)



## Implementation description
* Add annotations



## Steps to test
1. Under starter-code-cli folder run npm run dev -- -t
2. After selecting a combination, verify that the files are scrubbed properly and that docker-compose up runs successfully



## What should reviewers focus on?
* 


## Checklist
